DuckDuckGo/SettingsViewController.swift

```diff
@@ -195,7 +207,7 @@ class SettingsViewController: UITableViewController {
     }
 
     private func configureVersionText() {
-        versionText.text = versionProvider.localized
+        versionText.text = versionProvider.versionAndBuildNumber
```

Comment:
Just double checking this is a deliberate change (And is still localised?)?

Reply:
Yep, the new mocks have just the version number and not Name + Version number like we have today.
As for localization, it's using the same component as before, minus the name, see how `localized` was built before.

```
    public var localized: String {
        return "\(name) \(versionAndBuildNumber)"
    }
```